### DISCUSSION TITLE
Plugins / Add-ons

### DISCUSSION THREAD

**Discussion by SangeloDev** [outline/outline#3625] [Ideas]
Kind of a wild suggestion, but what if we had the ability to install plugins on our user accounts to extend functionality? Like more embeds, or maybe other integrations? That would really be cool, though I understand if something like that isn't currently possible.

**Comment by tommoor** [comment]
I think we need more integrations built directly into the product first to figure out where the boundaries of a plugin interface would be – eg what can plugins do, where can they insert UI etc. Right now it's unclear what that would look like and it would certainly be a lot of work to enable this in a secure way.
